charts/drax/charts/dashboard/values.yaml

```
@@ -54,7 +54,7 @@ config:
     - "accelleran.drax.4g.ric.raw.network_state"
     - "accelleran.drax.all.ric.processed.anr"
   # Defined for core-ui config.js
-  grafanaPort: 0
+  grafanaExternalPort: 0
```

There is no need to break this. Also it's external per definition from the dashboard's perspective as it's not included in there.

```suggestion
  grafanaPort: 0
```

---

Before we used the grafanaPort (30300) to allow our dRAX Dashboard to reach the Grafana Dashboard (so we were using external port)

But now with the introduction of grafana rest api calls in our Dashboard backend to handle alerts framework, I use some call that are using the internal grafana port service,

So I needed a distinction between the internal grafana port to be used inside of the cluster and the external grafana port to be used from outside the cluster,

Is there maybe a better approach?

---

But you want to use it for the backend then? Here it is underneath the comment that indicates it only being for the frontend. So we just need to introduce something twice (for both the front- and backend their own variable in their own config file, but the names can match as those are two different contexts).

---

Not sure what you mean, the external grafana port is used for front end (so under core-ui config.js) and the internal grafana port is used for backend (so under server config.js)
And both are configurable from drax values.yaml 	and then given to the Dashboard values.yaml to be used for the two config-map for front end and back end